### PR TITLE
Prepare release for version 0.4.0.

### PR DESCRIPTION
**Pull request by eeshangarg**
@timabbott: Could you please merge this? Thanks! :)

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## version 0.3.1
```
Comment (timabbott) [inline changelog.md]: Let's call this version 0.4.0, given what it does.

Comment (eeshangarg) [reply changelog.md]: Cool I'll do that :)

**Comment by eeshangarg** [comment]
@timabbott: Done! :)

**Comment by eeshangarg** [comment]
@timabbott, @edwardgeorge: Just made a PyPI release! :) Thanks!

**Pull request merged**